lib/src/html/html_generator_instance.dart

```diff
@@ -96,86 +96,78 @@ class HtmlGeneratorInstance implements HtmlOptions {
 
     generatePackage();
 
-    for (var lib in package.libraries) {
+    for (var lib in package.libraries.where((l) => l.isDocumented)) {
```

Comment:
Consider putting `isDocumented` onto Documentable, and having a utility method that would filter a list of Documentable objects to just the ones that were documented.

Reply:
(I see a lot of `.where((o) => o.isDocumented` below)

Reply:
done